docs/processors.md

```diff
@@ -12,6 +12,7 @@ Below is a list of supported processors with links to their documentation pages.
 | Group by Attributes Processor           | [groupbyattrsprocessor](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.66.0/processor/groupbyattrsprocessor/README.md) |
 | Group by Trace Processor                | [groupbytraceprocessor](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.66.0/processor/groupbytraceprocessor/README.md) |
 | Kubernetes Attributes Processor         | [k8sattributesprocessor](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.66.0/processor/k8sattributesprocessor/README.md) |
+| Log Count Processor                          | [logcountprocessor](../processor/logcountprocessor/README.md) |
```

Comment:
I debated on this. I went with Log Count, because it's the count of a log type, where the match expression is describing the type of log being counted.